src/rover/main.cpp

```diff
@@ -0,0 +1,174 @@
+//#define DEBUG
+#define TIME
+
+#include "camera.hpp"
+#include "../network/network.hpp"
+
+#include <cstddef>
+#include <iostream>
+#include <vector>
+#include <cstring>
+#include <cstdlib>
+#ifdef TIME
+	#include <chrono> // Time keeping from stdlib
+	using namespace std::chrono;
+#endif
+
+
+const unsigned int CAMERA_MESSAGE_SIZE = 65000;
+
+int main() {
+	/*
+	 * INITILIZATION: Rover Objects
+	 * OPTIMIZE: Consider making a rover class with an iteratable list of cameras
+	 */
+	unsigned int frame_counter = 0;
+	// Open all cameras
+	camera::CaptureSession session0;
```

Comment:
Can simply do 
```suggestion
	camera::CaptureSession session0, session1, session2, session3;
```

Reply:
I would prefer this to be an array of CaptureSessions (of length 4). That way, if we want to change the number of sessions later, we can do so easily. Even better would be to have a vector of CaptureSessions so that we dynamically load one for every connected camera on startup. This requires a lot more code though, so its just food for thought. What is here already is fine for now.